<!--- This file generated by `tools/apispec-rule-gen/main.go`. DO NOT EDIT --->

# azurerm_data_lake_store_invalid_encryption_state

Warns about values that appear to be invalid based on [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs).

Allowed values are:
- Enabled
- Disabled

## Example

```hcl
resource "azurerm_data_lake_store" "foo" {
  encryption_state = ... // invalid value
}
```

```
$ tflint
1 issue(s) found:

Error: "..." is an invalid value as encryption_state (azurerm_data_lake_store_invalid_encryption_state)

  on template.tf line 2:
  2:   encryption_state = ... // invalid value

Reference: https://github.com/terraform-linters/tflint-ruleset-azurerm/blob/v0.1.0/docs/rules/azurerm_data_lake_store_invalid_encryption_state.md

```

## Why

Requests containing invalid values will return an error when calling the API by `terraform apply`.

## How to Fix

Replace the warned value with a valid value.

## Source

This rule is automatically generated from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs). If you are uncertain about the warning, check the following API schema referenced by this rule.

https://github.com/Azure/azure-rest-api-specs/tree/master/specification/datalake-store/resource-manager/Microsoft.DataLakeStore/stable/2016-11-01/account.json
