<!--- This file generated by `tools/apispec-rule-gen/main.go`. DO NOT EDIT --->

# azurerm_data_factory_integration_runtime_managed_invalid_max_parallel_executions_per_node

Warns about values that appear to be invalid based on [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs).

The rule requires the value to be 1 or higher.

## Example

```hcl
resource "azurerm_data_factory_integration_runtime_managed" "foo" {
  max_parallel_executions_per_node = ... // invalid value
}
```

```
$ tflint
1 issue(s) found:

Error: max_parallel_executions_per_node must be 1 or higher (azurerm_data_factory_integration_runtime_managed_invalid_max_parallel_executions_per_node)

  on template.tf line 2:
  2:   max_parallel_executions_per_node = ... // invalid value

Reference: https://github.com/terraform-linters/tflint-ruleset-azurerm/blob/v0.1.0/docs/rules/azurerm_data_factory_integration_runtime_managed_invalid_max_parallel_executions_per_node.md

```

## Why

Requests containing invalid values will return an error when calling the API by `terraform apply`.

## How to Fix

Replace the warned value with a valid value.

## Source

This rule is automatically generated from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs). If you are uncertain about the warning, check the following API schema referenced by this rule.

https://github.com/Azure/azure-rest-api-specs/tree/master/specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json
