<!--- This file generated by `tools/apispec-rule-gen/main.go`. DO NOT EDIT --->

# azurerm_data_factory_linked_service_sql_server_invalid_data_factory_name

Warns about values that appear to be invalid based on [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs).

In this rule, the string must match the regular expression `^[A-Za-z0-9]+(?:-[A-Za-z0-9]+)*$``.

## Example

```hcl
resource "azurerm_data_factory_linked_service_sql_server" "foo" {
  data_factory_name = ... // invalid value
}
```

```
$ tflint
1 issue(s) found:

Error: "..." does not match valid pattern ^[A-Za-z0-9]+(?:-[A-Za-z0-9]+)*$ (azurerm_data_factory_linked_service_sql_server_invalid_data_factory_name)

  on template.tf line 2:
  2:   data_factory_name = ... // invalid value

Reference: https://github.com/terraform-linters/tflint-ruleset-azurerm/blob/v0.1.0/docs/rules/azurerm_data_factory_linked_service_sql_server_invalid_data_factory_name.md

```

## Why

Requests containing invalid values will return an error when calling the API by `terraform apply`.

## How to Fix

Replace the warned value with a valid value.

## Source

This rule is automatically generated from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs). If you are uncertain about the warning, check the following API schema referenced by this rule.

https://github.com/Azure/azure-rest-api-specs/tree/master/specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json
