<!--- This file generated by `tools/apispec-rule-gen/main.go`. DO NOT EDIT --->

# azurerm_mysql_server_invalid_ssl_enforcement

Warns about values that appear to be invalid based on [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs).

Allowed values are:
- Enabled
- Disabled

## Example

```hcl
resource "azurerm_mysql_server" "foo" {
  ssl_enforcement = ... // invalid value
}
```

```
$ tflint
1 issue(s) found:

Error: "..." is an invalid value as ssl_enforcement (azurerm_mysql_server_invalid_ssl_enforcement)

  on template.tf line 2:
  2:   ssl_enforcement = ... // invalid value

Reference: https://github.com/terraform-linters/tflint-ruleset-azurerm/blob/v0.1.0/docs/rules/azurerm_mysql_server_invalid_ssl_enforcement.md

```

## Why

Requests containing invalid values will return an error when calling the API by `terraform apply`.

## How to Fix

Replace the warned value with a valid value.

## Source

This rule is automatically generated from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs). If you are uncertain about the warning, check the following API schema referenced by this rule.

https://github.com/Azure/azure-rest-api-specs/tree/master/specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/mysql.json
