<!--- This file generated by `tools/apispec-rule-gen/main.go`. DO NOT EDIT --->

# azurerm_monitor_scheduled_query_rules_alert_invalid_severity

Warns about values that appear to be invalid based on [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs).

Allowed values are:
- 0
- 1
- 2
- 3
- 4

## Example

```hcl
resource "azurerm_monitor_scheduled_query_rules_alert" "foo" {
  severity = ... // invalid value
}
```

```
$ tflint
1 issue(s) found:

Error: "..." is an invalid value as severity (azurerm_monitor_scheduled_query_rules_alert_invalid_severity)

  on template.tf line 2:
  2:   severity = ... // invalid value

Reference: https://github.com/terraform-linters/tflint-ruleset-azurerm/blob/v0.1.0/docs/rules/azurerm_monitor_scheduled_query_rules_alert_invalid_severity.md

```

## Why

Requests containing invalid values will return an error when calling the API by `terraform apply`.

## How to Fix

Replace the warned value with a valid value.

## Source

This rule is automatically generated from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs). If you are uncertain about the warning, check the following API schema referenced by this rule.

https://github.com/Azure/azure-rest-api-specs/tree/master/specification/monitor/resource-manager/Microsoft.Insights/stable/2018-04-16/scheduledQueryRule_API.json
